src/content/docs/en/guides/deploy/vercel.mdx

```diff
@@ -27,18 +28,46 @@ To enable SSR in your Astro project and deploy on Vercel:
 
 Add [the Vercel adapter](/en/guides/integrations-guide/vercel/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
 
-```bash
-npx astro add vercel
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add vercel
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm dlx astro add vercel
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn dlx astro add vercel
+  ```
+  </Fragment>
+</PackageManagerTabs>
```

Comment:
`dlx` isn't required for pnpm/yarn. removing it to sync the syntax with our vercel integration guide.
``````suggestion
<PackageManagerTabs>
  <Fragment slot="npm">
  ```shell
  npx astro add vercel
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  pnpm astro add vercel
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  yarn astro add vercel
  ```
  </Fragment>
</PackageManagerTabs>
``````